### DISCUSSION TITLE
Prior upstream contributions to core ROS

### DISCUSSION THREAD

**Discussion by EzraBrooks** [space-ros/space-ros#85] [General]
I think it would be good to have a list of contributions to core ROS from Space ROS and the space community to help quantify the impact that safety-critical software development is having on core ROS.

**Comment by Bckempa** [comment]
https://github.com/ament/ament_lint/pull/419 will hopefully be an example we can use. As the list grows where should this be tracked/showcased, maybe a page on the website since that is easy to add to?
